examples/generics-actix/src/main.rs

```diff
@@ -0,0 +1,117 @@
+use std::{error::Error, net::Ipv4Addr};
+
+use actix_web::{
+    get, middleware::Logger, web::Json, web::Query, App, HttpServer, Responder, Result,
+};
+use serde::{Deserialize, Serialize};
+use utoipa::{
+    openapi::{schema::Object, schema::ObjectBuilder},
```

Comment:
can merge these two

Reply:
Do yo mean, that the `web::Json` and `web::Query` could be provided via `utoipa` as `utoipa::Json` and `utoipa::Query`?

Reply:
no. I was just noting the `use` syntax was not optimal.

Reply:
Ahh, you meant the imports. Sure they can.